Gemfile

```diff
@@ -34,19 +33,20 @@ gem 'sidekiq'
 
 # Services gems
 gem 'minigit', '~> 0.0.4'
-gem 'net-ssh', '~> 7.3.0'
+gem 'net-ssh', '~> 7.3'
 
 gem 'awesome_spawn',        '~> 1.6'
 gem 'default_value_for',    '~> 4.0'
-gem 'haml_lint',            '~> 0.51', :require => false
+gem 'haml_lint',            '~> 0.52', :require => false
 gem 'irb'
-gem 'manageiq-style',       '~> 1.5', ">=1.5.6", :require => false
+gem 'manageiq-style',       '~> 1.5.6',  :require => false
```

Comment:
This is not the same - think we want to keep the old way.

Reply:
yes. old says 1.6 is ok. new way says only 1.5.7